ext/search_kit/ang/crmSearchTasks/crmSearchTasks.component.js

```diff
@@ -42,8 +42,16 @@
         return $scope.$eval('' + ctrl.ids.length + action.number);
       };
 
+      this.updateActionData = function() {
+        if (this.entity === 'RelationshipCache') {
+          this.entity = this.entityInfo.title = 'Relationship';
+          this.entityInfo.title_plural = 'Relationships';
```

Comment:
`title` and `title_plural` need to go through `ts()`

Reply:
ah yes. Fixed 👍